modules/local/benchmarking/bed_to_csv/main.nf

```diff
@@ -0,0 +1,22 @@
+process BED_TO_CSV {
+    label "process_single"
+
+    conda "bioconda::pandas=1.5.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pandas:1.5.2' :
+        'biocontainers/pandas:1.5.2' }"
+
+    input:
+        tuple val(id), path(bedfile1), path(bedfile2) from ch_joined
```

Comment:
Named input channels are not common in nextflow DSL2, when calling `BED_TO_CSV(ch_joined)` the channel is passed to the first defined input automatically. If it would be necessary do do it like it was done here it would be hard to recycle the module.

Reply:
Basically just remove the `from ch_joined`